MAINTENANCE.md

```diff
@@ -0,0 +1,9 @@
+# Publishing to NPM
+
+So far there is little process to publish a new version of `@agoric/synthetic-chain` to NPM. (https://github.com/Agoric/agoric-3-proposals/issues/60 is to develop automation.)
+
+1. Bump the version number in `synthetic-chain/package.json`.
+2. PR to get it approved.
+3. When approved, run `npm publish` in the branch and merge.
```

Comment:
Shouldn't we also push a tag then? I don't remember if `npm publish` generates it.

Reply:
It's not generating tags. We had something making [tags for 0.0.{1,2,3}](https://github.com/Agoric/agoric-3-proposals/tags) but I don't recall what.

Reply:
And also a git tag, note that https://www.npmjs.com/package/@agoric/synthetic-chain?activeTab=versions includes versions that have no representation in https://github.com/Agoric/agoric-3-proposals/tags .